src/e2e/app/cli/repo-tests/src/commands/test_delete_command.rs

```diff
@@ -9,10 +9,16 @@
 
 use kamu_cli_e2e_common::{
     DATASET_DERIVATIVE_LEADERBOARD_SNAPSHOT_STR,
+    DATASET_ROOT_PLAYER_SCORES_INGEST_DATA_NDJSON_CHUNK_1,
```

Comment:
I'm not a big fan of reuse across test suites. In my experience it leads to dumping grounds of setup code that grow huge, hard to maintain, and fragile. Would caution against `kamu_api_server_client_ext.rs` becoming that (cc @s373r).

I think inlining test data or reusing at most on the test suite (module) level is better for readability and maintainability of tests.

Just sharing an opinion, not calling for action as part of this PR.

Reply:
I totally agree in general. But I don't think it will be much harm if we'll have "just a dataset" with "some" data chunks, for the cases where we don't check anything specific like transformations, template strings, etc.

Reply:
I'll duplicate a comment that is relevant to the thread, but for some reason fell out of it:

> > In my experience it leads to dumping grounds of setup code that grow huge, hard to maintain, and fragile
>
> I totally disagree. The reuse of test elements is great, hierarchy of test steps is great, not repeating yourself is gret. Tests are a 1st class citizen and have to be designed equally well as the main product, and in this case they don't become mess.

(c) @zaychenko-sergei, https://github.com/kamu-data/kamu-cli/pull/957#discussion_r1856225241